Journaley.Core/Journaley.Core.csproj

```diff
@@ -97,7 +97,9 @@
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
     <Error Condition="!Exists('..\packages\StyleCop.MSBuild.4.7.48.2\build\StyleCop.MSBuild.Targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\StyleCop.MSBuild.4.7.48.2\build\StyleCop.MSBuild.Targets'))" />
+    <Error Condition="!Exists('$(SolutionDir)\.nuget\NuGet.targets')" Text="$([System.String]::Format('$(ErrorText)', '$(SolutionDir)\.nuget\NuGet.targets'))" />
```

Comment:
No, you shouldn't enable that in the solution. In fact, it turns out these changes (in `Journaley.Core`, `Journaley`, `VersionExtractor`) broke the build on my side. Please remove these lines from all the affected projects.

You should just go to `Tools` -> `Options...` -> `NuGet Package Manager` -> `General` and check both the `Allow NuGet to download missing packages` and `Automatically check for missing pakcages during build in Visual Studio` options. The solution should build fine after enabling these options. If not, let's troubleshoot that together later on.